ivy/functional/extensions/__init__.py

```diff
@@ -19,8 +19,8 @@
 from .gradients import *
 from . import layers
```

Comment:
Actually `from . import <module>` inside this file is the problem. We should not be doing this at all, because then these will clash with those same imports in `ivy/functional/__init__.py`. Specifically, inside `ivy/functional/__init__.py` we have both `from .extensions import *` and also `from . import <module>`. This will duplicate `<module>` and one of them will be overwritten which should not happen.

Reply:
Makes sense! Solved in https://github.com/unifyai/ivy/pull/6128/commits/a6df22558acee1dc3d818521c90d07ba49b26ace